fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -2537,7 +2537,8 @@ public boolean canDisburse(final LocalDate actualDisbursementDate) {
                 LoanStatus.fromInt(this.loanStatus));
 
         boolean isMultiTrancheDisburse = false;
-        if (LoanStatus.fromInt(this.loanStatus).isActive() && isAllTranchesNotDisbursed()) {
+        if ((LoanStatus.fromInt(this.loanStatus).isActive() || LoanStatus.fromInt(this.loanStatus).isClosedObligationsMet()
```

Comment:
Couldn't we extract the LoanStatus.fromInt call before the if so it's not called for every condition? Would make the code much cleaner.